### ISSUE TITLE
NTR: Glyoxylic acid high in urine (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8384]
New term request Glyoxylic acid high in urine (elthat_150420023459): Glyoxylate

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000702
- label: Elevated urinary glyoxylic acid level
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: PMID:449695
- def: The amount of glyoxylic acid in the urine, normalized for urine concentration, is above the upper limit of normal. The conjugate base of glyoxylic acid is known as glyoxylate.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8384
- synonyms: Elevated urinary glyoxylate level(NONE)|Elevated urinary alpha-ketoacetic acid level(NONE)|Elevated urinary oxoacetic acid level(NONE)
